clustpy/deep/enrc.py

```diff
@@ -800,15 +815,16 @@ def enrc_encode_decode_batchwise_with_loss(V: torch.Tensor, centers: list, model
             embeddings.append(z_rot.detach().cpu())
             z_rot_back = _rotate_back(z_rot=z_rot, V=V)
             reconstruction = model.decode(z_rot_back)
```

Comment:
this would not work for every model. only for autoencoders, which have the `decode` function. maybe we can leave it for now that ENRC can only be used with the abstract autoencoder class? Otherwise the code would need some restructuring. I can do that, but not sure when I will have the time.